Unix/scripts/config_keytab_update.sh

```diff
@@ -58,6 +63,10 @@ unconfigure()
     grep -v "$omikeytab" $tmpfile >$tmpfile2 
     crontab $tmpfile2
 
+    # clean temp file
+    cleanTempFile
```

Comment:
You don't need to call cleanTempFile here, since it's also called outside function.

Reply:
good catch, will fix it, thanks.